2019/Makefile

```diff
@@ -1,6 +1,9 @@
 ifeq (,$(OUT))
 $(error you must define OUT)
 endif
+ifeq (,$(DMD))
+$(error you must define DMD)
+endif
```

Comment:
Not sure what the reason for not doing a nice default `DMD=dmd` was, but at least let's warn the users here.